src/cluster.c

```diff
@@ -4383,6 +4383,24 @@ int getSlotOrReply(client *c, robj *o) {
     return (int) slot;
 }
 
+/* Returns an indication if the replica node is fully available
+* and should be listed in CLUSTER SLOTS response.
+* Returns 1 for available nodes, 0 for nodes that have 
+* not finished their initial sync, in failed state, or are 
+* otherwise considered not available to serve read commands. */
```

Comment:
```suggestion
/* Returns an indication if the replica node is fully available
 * and should be listed in CLUSTER SLOTS response.
 * Returns 1 for available nodes, 0 for nodes that have 
 * not finished their initial sync, in failed state, or are 
 * otherwise considered not available to serve read commands. */
```